src/openqdc/datasets/ani.py

```diff
@@ -50,6 +60,25 @@ def read_raw_entries(self):
         samples = read_qc_archive_h5(raw_path, self.__name__, self.energy_target_names, self.force_target_names)
         return samples
 
+    @property
```

Comment:
why these stats are not computed on the fly? using numpy /torch or at loading time? It would take less than 15s for most datasets no? Doing this way will create problem if the data change.

Reply:
Because the on the fly computation with the original script was too slow for some datasets.